admin/backend/adapters/base/resource.js

```diff
@@ -80,6 +80,10 @@ class BaseResource {
     this._decorated = this._decorated || new this._Decorator(this)
     return this._decorated
   }
+
+  databaseType() {
```

Comment:
please write jsdoc - I added lots of them in separate repo so right now when we create new features we have to add docs to methods which will be treated as an interface for developers